### ISSUE TITLE
History of vasectomy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10098]
**Preferred term label:**
History of vasectomy

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0455564

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Post-vasectomy pain syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000063
- label: History of vasectomy
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: A medical history of having undergone a vasectomy prior to the current presentation.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10098
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
HPO ID corrected to  HP:0040410
